project/MimaExcludes.scala

```diff
@@ -36,6 +36,10 @@ object MimaExcludes {
 
   // Exclude rules for 3.3.x from 3.2.0 after 3.2.0 release
   lazy val v33excludes = v32excludes ++ Seq(
+    ProblemFilters.exclude[IncompatibleResultTypeProblem]("org.apache.spark.ml.param.FloatParam.jValueEncode"),
+    ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.ml.param.FloatParam.jValueDecode"),
+    ProblemFilters.exclude[IncompatibleMethTypeProblem]("org.apache.spark.mllib.tree.model.TreeEnsembleModel#SaveLoadV1_0.readMetadata"),
+    ProblemFilters.exclude[IncompatibleResultTypeProblem]("org.apache.spark.sql.expressions.MutableAggregationBuffer.jsonValue")
```

Comment:
This seems to be used in our API. Is this incompatibility okay?

```
  /**
   * Initializes the given aggregation buffer, i.e. the zero value of the aggregation buffer.
   *
   * The contract should be that applying the merge function on two initial buffers should just
   * return the initial buffer itself, i.e.
   * `merge(initialBuffer, initialBuffer)` should equal `initialBuffer`.
   *
   * @since 1.5.0
   */
  def initialize(buffer: MutableAggregationBuffer): Unit
```

cc @HyukjinKwon , @cloud-fan

Reply:
`MutableAggregationBuffer.jsonValue` is marked as `private[sql]` so I thought it's OK. But if I'm missing something, please let me know.
Maybe, users need to re-compile their applications due to the binary compatibility of `MutableAggregationBuffer` will break?